eth-contracts/test/serviceProvider.test.js

```diff
@@ -186,8 +229,8 @@ contract('ServiceProvider test', async (accounts) => {
       assert.equal(regTx.stakedAmountInt, DEFAULT_AMOUNT)
 
       // Confirm balance updated for tokens
-      let finalBal = await getTokenBalance(token, stakerAccount)
-      assert.equal(initialBal, finalBal + DEFAULT_AMOUNT, 'Expect funds to be transferred')
+      let finalBal = await token.balanceOf(stakerAccount)
+      assert.isTrue(initialBal.eq(finalBal.add(DEFAULT_AMOUNT)), 'Expect funds to be transferred')
```

Comment:
where possible, tried to use big number evaluation instead of comparing potentially unknown types. worth revisiting across this ethContracts test suite overall